core/controllers/creator_dashboard_test.py

```diff
@@ -70,12 +70,6 @@ def _get_continuous_computation_class(cls):
 
 class HomePageTests(test_utils.GenericTestBase):
 
-    def test_logged_out_homepage(self):
```

Comment:
This is a better place for this test (As per the test class name)

Reply:
Done.